docs/_includes/getting-started.html

```diff
@@ -2,29 +2,7 @@
     <div class="centered" style="padding-top: 32px">
         <h1 id="getting-started">Getting Started</h1>
         <p>
-            Using Dotty to compile your project is now pretty simple. Create the
-            following structure:
-        </p>
-        <pre class="sourceCode bordered">
-<code>.
-├── build.sbt
-├── project
-│   ├── build.properties
-│   └── plugins.sbt
-└── src</code></pre>
-        <h2 id="build.sbt">build.sbt</h2>
-        <pre class="sourceCode bordered"><code>name := "application"
-version := "0.1"
-enablePlugins(DottyPlugin)</code></pre>
-        <h2 id="build.properties">build.properties</h2>
-        <pre class="sourceCode bordered"><code>sbt.version=0.13.11</code></pre>
-        <h2 id="plugins.sbt">plugins.sbt</h2>
-        <pre class="sourceCode bordered"><code>addSbtPlugin("com.felixmulder" % "sbt-dotty" % "0.1.7")</code></pre>
-        <p>
-            This plugin is based on the
-            <a href="https://github.com/smarter/dotty-example-project">
-                dotty-example-project
-            </a>, but let’s you skip building dotty from scratch.
+            See the <a href="https://github.com/lampepfl/dotty-example-project">example project</a>.
```

Comment:
> Instead of "google dotty, find webpage, find link to example project, read most of README to know which options to use, set up sbt config, run dotty".

We could make an `sbt new` template and add the command to create it on the homepage: http://www.scala-sbt.org/0.13/docs/sbt-new-and-Templates.html